cpp/src/barretenberg/stdlib/merkle_tree/merkle_tree.cpp

```diff
@@ -133,6 +133,65 @@ template <typename Store> fr_hash_path MerkleTree<Store>::get_hash_path(index_t
     return path;
 }
 
+template <typename Store> fr_sibling_path MerkleTree<Store>::get_sibling_path(index_t index)
+{
+    fr_sibling_path path(depth_);
+
+    std::vector<uint8_t> data;
+    bool status = store_.get(root().to_buffer(), data);
+
+    for (size_t i = depth_ - 1; i < depth_; --i) {
+        if (!status) {
+            // This is an empty subtree. Fill in zero value.
+            path[i] = zero_hashes_[i];
+            continue;
+        }
+
+        if (data.size() == 64) {
```

Comment:
whats special about 64?

Reply:
Hello, I intended this PR to be reviewed and merged after https://github.com/AztecProtocol/barretenberg/pull/584 is merged. This PR is unfinished and messy.

But your comment applies to #584 as well. The get_sibling_path method is based on get_hash_path and in both cases 64 is a size of a regular node (65 is "stump"). I think introducing a constant is a good idea and I will do that in #584.